Gemfile

```diff
@@ -28,11 +27,11 @@ gem 'amistad'
 
 group :development, :test do
   gem 'coveralls', require: false
-  gem 'simplecov', github: 'AgileVentures/simplecov'
+  gem 'simplecov'# ,github: 'AgileVentures/simplecov'
```

Comment:
I think we can remove the comment here unless there's any specific reason for keeping this around.